e2e/tests/functional/plugins/timeConductor/timeConductor.e2e.spec.js

```diff
@@ -281,29 +281,3 @@ test.describe('Global Time Conductor', () => {
     // select an option and verify the offsets are updated correctly
   });
 });
-
-test.describe('Time Conductor History', () => {
```

Comment:
Is this totally gone?

Reply:
it's no longer relevant because you can't set milliseconds in the time conductor anymore. The time conductor history menu is still there, though. Maybe we should keep the suite